src/overlays/actors/ovl_En_Part/z_en_part.c

```diff
@@ -28,16 +30,114 @@ const ActorInit En_Part_InitVars = {
     (ActorFunc)EnPart_Draw,
 };
 
-#endif
+void EnPart_Init(Actor* thisx, PlayState* play) {
+}
+
+void EnPart_Destroy(Actor* thisx, PlayState* play) {
+}
+
+void func_80865390(EnPart* this, PlayState* play) {
+    f32 randVar;
+
+    this->unk144 = 1;
+    this->actor.world.rot.y = (Rand_ZeroOne() * 20000.0f);
+    switch (this->actor.params) {
+        case PARAMS_1:
+        case PARAMS_4:
+            randVar = Rand_ZeroOne() * 17.0f;
+            this->unk146 += (s16)randVar + 5;
```

Comment:
Looks like this is the only use of `randVar` is it needed to match?

Reply:
I'd like an answer on this one way or the other.

Reply:
Hm, I thought it was needed but turns out I was wrong. Will change this (+ params enum name)